ons_alpha/jinja2/templates/pages/topics/topic_page.html

```diff
@@ -1 +1,45 @@
 {% extends "templates/pages/index_page.html" %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% block main %}
+    <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+        <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+            <section id="summary">
+                <h2>{{ _("Summary")}}</h2>
+                <p>{{ page.summary}}</p>
+            </section>
+        </div>
+    </div>
+
+    {% block preSubpages %}
+    <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+        <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+            <section id="topic_page_nav">
+                {% for key in subpage_dict.keys() %}
+                    <a href="#{{ key }}">{{ key }}</a>
+                {% endfor %}
+            </section>
+        </div>
+    </div>
+    {% endblock %}
+
+    {% block index %}
+        <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+            <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+                {% if subpages %}
+                    <h2>{{"Index"}}</h2>
+                    {% for subpage_key in subpage_dict.keys() %}
+                        <h3 id="{{ subpage_key }}">{{ subpage_key }}</h3>
+                        {% for subpage_value in subpage_dict[subpage_key]  %}
+                            <a href="{{ subpage_value.pageurl }}">
```

Comment:
Not quite:

```suggestion
                            <a href="{% pageurl subpage_value %}">
```
https://docs.wagtail.org/en/stable/topics/writing_templates.html#pageurl

There are also other places where `.url` is used directly, which should be updated.